NEWS.md

```diff
@@ -4,6 +4,8 @@
 
 * `show_progress()` uses `rlang::is_interactive()` instead of `base::interactive()` (#1356).
 
+* `read_builtin()` does more argument checking, so that we catch obviously malformed input before passing along to `utils::data()`.
```

Comment:
It is kind of a chicken and egg problem, but if you don't have an issue open for this problem then I tend to open the PR, document the issue in the PR like you have done, then do another commit that would add `(#1361)` to the NEWS bullet. It is annoying that you can't do that on a first pass, but you don't know the PR number (for sure) until you open it.